orca-api/src/main/java/com/netflix/spinnaker/orca/api/pipeline/ExecutionPreprocessor.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Netflix, Inc.
+ * Copyright 2021 Armory
```

Comment:
I'm not exactly sure what the rules are here but my first thought was that this should not change (2019 -> 2021 seems OK though). 🤷

Reply:
Cool happy to leave this. 👍